presto-main/src/main/java/com/facebook/presto/memory/MemoryManagerConfig.java

```diff
@@ -34,6 +35,7 @@
     private DataSize maxQueryMemoryPerNode = new DataSize(1, GIGABYTE);
     private boolean killOnOutOfMemory;
     private Duration killOnOutOfMemoryDelay = new Duration(5, MINUTES);
+    private Duration maxQueryCpuTime = new Duration(100, DAYS);
```

Comment:
Make this `Long.MAX_VALUE, NANOSECONDS`. 100 days is half an hour in a large cluster. It's too small.